crates/bevy_tasks/src/task_pool.rs

```diff
@@ -102,9 +102,6 @@ impl TaskPool {
                 let ex = Arc::clone(&executor);
                 let shutdown_rx = shutdown_rx.clone();
 
-                // miri does not support setting thread names
-                // TODO: change back when https://github.com/rust-lang/miri/issues/1717 is fixed
-                #[cfg(not(miri))]
                 let mut thread_builder = {
```

Comment:
I don't think we'd have used a block expression here if it weren't for the `cfg`